http_types/utils.py

```diff
@@ -179,6 +198,11 @@ def from_dict(obj: Any) -> Response:
             body_as_json = parse_body(obj_copy['body'])
             obj_copy['bodyAsJson'] = body_as_json
 
+        if "timestamp" in obj_copy:
+            obj_copy['timestamp'] = datetime.fromisoformat(obj_copy['timestamp'])
+        else:
+            obj_copy['timestamp'] = None
+
         res = Response(**obj_copy)
         ResponseBuilder.validate(res)
```

Comment:
This uses run-time type-checking with `typeguard` and that could become a problem because it doesn't understand mixing `total=True` and `total=False`, we may have to remove this for now.